assignments/objects.js

```diff
@@ -49,16 +101,24 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
-
-// Log the parent object's name
-
-// Log the child's age
-
-// Log the name and age of the grandchild
-
-// Have the parent speak
-
-// Have the child speak
-
-// Have the grandchild speak
+const parent = {
+  name: "Susan",
+  age: 70,
+  speak: function(){
+    console.log(`Hi I'm ${this.name}`);
+  },
+  child: {
+    name: "George",
+    age: 50,
+    speak: function(){
+      console.log(`Hi I'm ${this.name}`);
+    },
+    grandchild: {
+      name: "Sam",
+      age: 30,
+      speak: function(){
+        console.log(`Hi I'm ${this.name}`);
+      }
+    }
+  }
+}
```

Comment:
`console.log` the individual properties and invoke the corresponding methods of the parent object

```js
// Log the child's age

// Log the name and age of the grandchild

// Have the parent speak

// Have the child speak

// Have the grandchild speak
```